semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/structure/TypeDecl.scala

```diff
@@ -112,21 +111,12 @@ class TypeDecl(val wrapped: NodeSteps[nodes.TypeDecl]) extends AnyVal {
     * If this is an alias type declaration, go to its underlying type declaration
     * else unchanged.
     */
-  def unravelAlias: NodeSteps[nodes.TypeDecl] = {
-    new NodeSteps(raw.map { typeDecl =>
+  def unravelAlias: NodeSteps[nodes.TypeDecl] =
+    wrapped.map { typeDecl =>
       if (typeDecl.aliasTypeFullName.isDefined) {
-        typeDecl
-          .vertices(Direction.OUT, EdgeTypes.ALIAS_OF)
-          .next
-          .asInstanceOf[nodes.Type]
-          .vertices(Direction.OUT, EdgeTypes.REF)
-          .next
-          .asInstanceOf[nodes.TypeDecl]
-      } else {
-        typeDecl
-      }
-    })
-  }
+        typeDecl.aliasOfOut.next.refOut.next
+      } else typeDecl
```

Comment:
If the `if` block has bracket PLEASE do also use brackets for the `else` block. This is just asking for bugs.

Reply:
really? ok. removed curlies from the `if` block now as well since it's very short